unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTFast/Wrappers/GLTFTextureDownloaderWrapper.cs

```diff
@@ -27,7 +27,8 @@ public Texture2D GetTexture(bool linear)
         {
             Texture2D texture2D = new Texture2D(1, 1, TextureFormat.RGBA32, 0, linear);
 
-            if (asyncOp.webRequest.downloadHandler.data != null)
+            if (asyncOp.webRequest != null && asyncOp.webRequest.downloadHandler != null
+                                           && asyncOp.webRequest.downloadHandler.data != null)
```

Comment:
FWIW, maybe also test for `asyncOp`.
```suggestion
            if (asyncOp?.webRequest?.downloadHandler?.data != null)
```

Reply:
Added suggestion, and divided even further to learn more about the crash